dash/backend/src/modules/exceptions/dao/exceptions.dao.ts

```diff
@@ -193,7 +193,8 @@ export class ExceptionsDao {
 
         //console.log('getAllFilteredPolicyExceptions: ', sql.toQuery());
 
-        return await knexnest(sql).then(data => data);
+        const result = await knexnest(sql).then(data => data);
```

Comment:
is the .then() necessary?

Reply:
elsewhere I've seen the .then() use the plainToObject() to cast into the appropriate object class.

Reply:
.then(data => plainToClass(data, ExceptionQueryDto)